plugins/style/Buttons.css

```diff
@@ -16,7 +16,7 @@ button.map-button {
     cursor: pointer;
 }
 
-button.map-button:disabled {
+button.map-button.locate-button-DISABLED {
```

Comment:
This looks like an accidental commit?

Reply:
Not really... I recognized that the map locator button didn't get transparent, if I block the location within the browser. Than I checked the css class of the button and it is named "locate-button-DISABLED". So I changed this within the Buttons.css.

Reply:
Ah ok!

Reply:
It hasn't anything to do with the original commit here, that's right. But as it is just a small change, I wouldn't start another pull request for it.